pkg/ambex/main.go

```diff
@@ -192,6 +196,14 @@ func parseArgs(ctx context.Context, rawArgs ...string) (*Args, error) {
 		dlog.Errorf(ctx, "Invalid AMBASSADOR_AMBEX_SNAPSHOT_COUNT: %s, using %d", numsnapStr, args.numsnaps)
 	}
 
+	// edsByPass will bypass using EDS and will insert the endpoints into the cluster data manually
+	// This is a stop gap solution to resolve 503s on certification rotation
+	edsByPass := os.Getenv("AMBASSADOR_EDS_BY_PASS")
+	if strings.ToLower(edsByPass) == "true" {
```

Comment:
This should be equivalent to `strconv.ParseBool`, but in general: Prefer `strconv.ParseBool`.